jormungandr/src/blockchain/storage.rs

```diff
@@ -101,6 +101,17 @@ impl Storage {
         }
     }
 
+    pub fn get_storage_block_info(
```

Comment:
Bikeshed: could just name `get_block_info`.

Reply:
Just removed it, it is not used anyway.

Reply:
Not sure how this passed the linting steps BTW.